src/task/runtime.rs

```diff
@@ -175,12 +174,9 @@ impl Runtime {
         #[cfg(feature = "time")]
         let timeout = self.timer_runtime.borrow().min_timeout();
 
-        const UNINIT_ENTRY: MaybeUninit<Entry> = MaybeUninit::uninit();
-        let mut entries = [UNINIT_ENTRY; 16];
-        match self.driver.borrow_mut().poll(timeout, &mut entries) {
-            Ok(len) => {
-                for entry in &mut entries[..len] {
-                    let entry = unsafe { std::mem::replace(entry, UNINIT_ENTRY).assume_init() };
+        match self.driver.borrow_mut().poll_entries::<1024>(timeout) {
```

Comment:
This is like 24 * 1024 bytes per poll, passed by value, on stack...

Reply:
Revert back to 16 then:)

Reply:
Can we provide a way to to avoid double write for io-uring & mio.

If we insist on having entries param in the signature it's possible to provide empty entries slice and fill only completion queue.

Access to the completions queue could be provided only for io-uring and mio.

Reply:
Alternatively it's possible to exclude completions queue from all drivers and support a generic entries container that supports `Extend` and push_back.

If runtime guarantees upper bound for max completed entries it could provide non growable container. If a client runtime can't process all completed items by some reason it could use VecDeque to maintain progress.

Reply:
> There's no double write if the user provided entries param is large enough.

I see.

Then let's create an empty completion queue in mio/io_uring. If runtime can bound the number of outstanding io operations then the queue will never be allocated.

Reply:
> Revert back to 16 then:)

Can we include a mutable uninited entries array in poll interface? Then driver client could choose the type of allocation (stack vs heap)?

Reply:
> Alternatively it's possible to exclude completions queue from all drivers and support a generic entries container that supports `Extend` and push_back.

I'll implement this.